src/DotLiquid/StandardFilters.cs

```diff
@@ -9,6 +9,7 @@
 using System.Reflection;
 using System.Threading;
 using DotLiquid.Util;
+using System.Text;

```

Comment:
Consider adding the new using in alphabetical order with the existing System using statements for readability and avoid duplicate entries when the list becomes really long.